config/suit/templates/nrf54h20/default/v1/root_with_binary_nordic_top.yaml.jinja2

```diff
@@ -136,36 +140,64 @@ SUIT_Envelope_Tagged:
     suit-install:
     - suit-directive-set-component-index: 0
 {%- if radio is defined %}
-    - suit-directive-override-parameters:
-        suit-parameter-uri: '#{{ radio['name'] }}'
-    - suit-directive-fetch:
-      - suit-send-record-failure
-    - suit-condition-dependency-integrity:
-      - suit-send-record-success
-      - suit-send-record-failure
-      - suit-send-sysinfo-success
-      - suit-send-sysinfo-failure
-    - suit-directive-process-dependency:
-      - suit-send-record-success
-      - suit-send-record-failure
-      - suit-send-sysinfo-success
-      - suit-send-sysinfo-failure
+    - suit-directive-run-sequence:
```

Comment:
Please add a comment, that since `update-candidate` sequence has ensured that the update candidate includes a compatible set of manifests, we may blindly process whatever is attached in the `suit-install` sequence.

Reply:
Ok, added - see if it's OK